test-runner/src/main/scala/com/typesafe/spark/test/mesos/framework/runners/MultiClusterModeRunner.scala

```diff
@@ -0,0 +1,129 @@
+package com.typesafe.spark.test.mesos.framework.runners
+
+import java.io.File
+import java.net.URL
+
+import Utils._
+import com.typesafe.config.{Config, ConfigFactory}
+import com.typesafe.spark.test.mesos.mesosstate.MesosCluster
+
+import org.apache.zookeeper.{ZKUtil, ZooKeeper, KeeperException}
+
+import scala.sys.process.Process
+
+object MultiClusterModeRunner {
```

Comment:
To avoid creating a separate runner, can we combine this with ClusterModeRunner?

If we actually care to run single-dispatcher tests (I don't see why we do, though), then we could introduce a --two-dispatcher parameter to the existing `mit` task.